primitives/account/Cargo.toml

```diff
@@ -25,14 +27,16 @@ sp-std = { workspace = true }
 [features]
 default = ["std"]
 std = [
-	"impl-serde/std",
 	"hex/std",
+	"impl-serde/std",
```

Comment:
root Cargo.toml

```toml
impl-serde = { version = "0.4.0", default-features = false }
```

so we need to enable std feature of impl-serde when use std feature of fp-account

Reply:
Yes, if we use impl-serde in the std feature of fp-account, we also need to enable std feature of impl-serde, because the impl-serde doesn't enable the std feature in the root Cargo.toml